event_sale_session/models/event_event.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2017-19 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class EventEvent(models.Model):
+    _inherit = "event.event"
+
+    order_line_ids = fields.One2many(
+        comodel_name="sale.order.line",
+        inverse_name="event_id",
+        string="Sales Order Lines",
+    )
+    unconfirmed_qty = fields.Integer(
+        compute="_compute_unconfirmed_qty",
+        store=True,
+    )
+
+    @api.depends(
+        "order_line_ids",
+        "order_line_ids.product_uom_qty",
+        "order_line_ids.order_id.state",
+    )
+    def _compute_unconfirmed_qty(self):
+        for event in self:
+            event.unconfirmed_qty = int(
+                sum(
+                    event.order_line_ids.filtered(
+                        lambda x: x.order_id.state in ("draft", "sent")
+                    ).mapped("product_uom_qty")
+                )
+            )
+
+    def button_open_unconfirmed_event_order(self):
+        action = self.env["ir.actions.act_window"]._for_xml_id("sale.action_quotations")
+        sales = (
+            self.env["sale.order.line"]
+            .search([("event_id", "in", self.ids)])
+            .mapped("order_id")
+            .ids
+        )
+        action["domain"] = [("id", "in", sales), ("state", "in", ("draft", "sent"))]
+        action["context"] = {}
+        return action
```

Comment:
This stuff doesn't belong to this module (I know it was, but it wasn't right).

Reply:
Our customer needs the information to check from event form, that's why is necessary.

Reply:
I know: but that doesn't mean that this code belongs here :smile It should either go to its own module or to custom code

Reply:
I agree this "unconfirmed quantity" feature doesn't belong here. This module's purpose should be to implement the event sessions with at most the same features you find in standard Events module, not more.

Any other extra feature should be moved to a separate module. (e.g: `event_sale_session_unconfirmed_qty`? notice you could even have a `event_sale_unconfimed_qty` module that implements this in standard events, because the feature doesn't exist in standard)